src/main/java/com/example/javaee_jpa_hibernate/model/Invoice.java

```diff
@@ -2,7 +2,6 @@
 
 import com.example.javaee_jpa_hibernate.model.counterparty.Counterparty;
 import com.example.javaee_jpa_hibernate.model.invoice_item.InvoiceItem;
-import jakarta.persistence.Entity;
```

Comment:
This is the main reason why it didn't want to work - you have used the wrong import: `jakarta.persistence.Entity` instead of `javax.persistence.Entity` (which was already added)